tests/airport.spec.js

```diff
@@ -0,0 +1,303 @@
+const { assertEquals } = require('../testing-framework');
+const Airport = require(`../src/Airport.js`);
+const Plane = require(`../src/Plane.js`);
+
+let expected;
+let actual;
+let result;
+let airport;
+let plane;
+let plane2;
+let plane3;
+let plane4;
+let plane5;
+let plane6;
```

Comment:
I still haven't fully grasped how the instance of the plane class can work here so there was likely a better way of testing multiple instances of the plane class without creating separate planes like this for test 6.